plugin/trino-ignite/src/main/java/io/trino/plugin/ignite/IgniteJdbcClient.java

```diff
@@ -0,0 +1,562 @@
+//                throw new TableNotFoundException(
+//                        schemaTableName,
+//                        format("Table '%s' has no supported columns (all %s columns are not supported)", schemaTableName, allColumns));
+//            }
+//            return ImmutableList.copyOf(columns);
+//        }
+//        catch (SQLException e) {
+//            throw new TrinoException(JDBC_ERROR, e);
+//        }
+//    }
+//
+//    private static RemoteTableName getRemoteTable(ResultSet resultSet)
+//            throws SQLException
+//    {
+//        return new RemoteTableName(
+//                Optional.ofNullable(resultSet.getString("TABLE_CAT")),
+//                Optional.ofNullable(resultSet.getString("TABLE_SCHEM")),
+//                resultSet.getString("TABLE_NAME"));
+//    }
+
+    @Override
+    public JdbcOutputTableHandle beginInsertTable(ConnectorSession session, JdbcTableHandle tableHandle, List<JdbcColumnHandle> columns)
+    {
+        SchemaTableName schemaTableName = tableHandle.asPlainTable().getSchemaTableName();
+        JdbcIdentity identity = JdbcIdentity.from(session);
+
+        try (Connection connection = connectionFactory.openConnection(session)) {
+            String remoteSchema = getIdentifierMapping().toRemoteSchemaName(identity, connection, schemaTableName.getSchemaName());
+            String remoteTable = getIdentifierMapping().toRemoteTableName(identity, connection, remoteSchema, schemaTableName.getTableName());
+            String catalog = connection.getCatalog();
+
+            ImmutableList.Builder<String> columnNames = ImmutableList.builder();
+            ImmutableList.Builder<Type> columnTypes = ImmutableList.builder();
+            ImmutableList.Builder<JdbcTypeHandle> jdbcColumnTypes = ImmutableList.builder();
+            for (JdbcColumnHandle column : columns) {
+                columnNames.add(column.getColumnName());
+                columnTypes.add(column.getColumnType());
+                jdbcColumnTypes.add(column.getJdbcTypeHandle());
+            }
+
+            return new JdbcOutputTableHandle(
+                    catalog,
+                    remoteSchema,
+                    remoteTable,
+                    columnNames.build(),
+                    columnTypes.build(),
+                    Optional.of(jdbcColumnTypes.build()),
+                    remoteTable);
+        }
+        catch (SQLException e) {
+            throw new TrinoException(JDBC_ERROR, e);
+        }
+    }
+
+    @Override
+    public void finishInsertTable(ConnectorSession session, JdbcOutputTableHandle handle)
+    {
+    }
+
+    @Override
+    public void rollbackCreateTable(ConnectorSession session, JdbcOutputTableHandle handle)
+    {
+        // avoid delete source table in Ignite
+        if (handle.getTableName().equals(handle.getTemporaryTableName())) {
+            return;
```

Comment:
I'd understand this for insert case, but not for create table case.

anyway, we should make inserts in Ingnite connector follow the standard path (see comment above) and thus we would not need to override finishInsertTable and rollbackCreateTable